notice/cn/2020-04-02-Inflation-Bug-Disclosure.html

```diff
@@ -163,7 +163,7 @@ <h2>新的协议内容</h2>
 <section class="notice default"><div class="section-wrapper"><div>
   <h2>Thanks</h2>
   <p>
-    This document was translated into Chinese by <code>ludoviso/</code> and <code>ids/</code> from the hnsfans.com community.
+    This document was translated into Chinese by < a href=' '>ludoviso/</ a> and < a href='http://ids.hns.to'>ids/</ a> from the < a href='https://www.hnsfans.com/'>hnsfans.com</ a> community.
```

Comment:
I think it's OK to link to `hnsfans.com` since on the homepage we do also have links for telegram chat and IRC channels. I'd prefer not use `hns.to` links however, since it is an insecure gateway. Do `ludoviso` and/or `ids` have github profiles?

Reply:
sure，thanks！
 https://github.com/v1xingyue  is for ids
 ludoviso has no github, so....... never mind :)

Reply:
Ok thank you for understanding, and thank you so so much for the translation. I've up dated the typo and merged the hnsfans link and github user link. <3